node/chainSimulator/components/testOnlyProcessingNode.go

```diff
@@ -292,3 +295,51 @@ func (node *testOnlyProcessingNode) createNodesCoordinator(pref config.Preferenc
 
 	return nil
 }
+
+func (node *testOnlyProcessingNode) ProcessBlock(nonce uint64, round uint64) error {
```

Comment:
missing comment
Rename this to "CreateNewBlock" ?

Reply:
renamed.